docs/idl/NC-Framework.webidl

```diff
@@ -5,11 +5,11 @@ $macro(HeaderComments)
 	//	NC-Framework classes and datatypes
```

Comment:
Maybe somewhere in these opening comments we should document the naming conventions.  Just an idea.

Reply:
We will tackle as a separate point by adding wording in the Markdown.